plugins/org.jkiss.dbeaver.ui.statistics/src/org/jkiss/dbeaver/ui/statistics/UIStatisticsActivator.java

```diff
@@ -110,6 +109,9 @@ public static ImageDescriptor getImageDescriptor(String path) {
     }
 
     public DBPPreferenceStore getPreferences() {
+        if (preferences == null) {
```

Comment:
Why? It must be instantiated on bundle activation

Reply:
Generally, avoiding hanging bundle on start and case preferences reset (remove pref files) is better to create preferences at first call.